src/Test/Perf/Utilities/Logger.cs

```diff
@@ -5,24 +5,50 @@
 
 namespace Roslyn.Test.Performance.Utilities
 {
+    /// <summary>
+    /// An interface for logging messages.  A global ILogger implementation
+    /// exists at Roslyn.Test.Performance.Utilities.RuntimeSettings.logger.
+    /// </summary>
     public interface ILogger
     {
+        /// <summary>
+        /// Logs a string through the logger.
+        /// </summary>
+        /// <param name="v"></param>
         void Log(string v);
+
+        /// <summary>
+        /// Flushes the cache (if one exists)
+        /// </summary>
         void Flush();
     }
+
+    /// <summary>
+    /// An implementation of ILogger that prints to the console, and also
+    /// writes to a file.
+    /// </summary>
     public class ConsoleAndFileLogger : ILogger
     {
         private readonly string _file;
         private readonly StringBuilder _buffer = new StringBuilder();
 
+        /// <summary>
+        /// Constructs a new ConsoleAndFileLogger with a default log 
+        /// file of 'log.txt'.
+        /// </summary>
+        /// <param name="file"></param>
         public ConsoleAndFileLogger(string file = "log.txt")
```

Comment:
Consider private (explicit) interface implementation and removing the doc comments here (presumably, nobody should call directly through the ```ConsoleAndFileLogger```).